addons/analytic/tests/test_analytic_account.py

```diff
@@ -29,6 +29,14 @@ def setUpClass(cls):
         ])
         cls.company_b_branch = cls.env['res.company'].create({'name': "B Branch", 'parent_id': cls.company.id})
 
+        """ Removes access rights linked to timesheet and project as these add
+        record rules blocking analytic flows; account overrides it"""
+        if 'account.account' not in cls.env:
```

Comment:
This could probably be moved in the `setup_independent_user` maybe before the effective creation so we don't link -> unlink